ui/apps/platform/src/Containers/Vulnerabilities/ExceptionManagement/ExceptionRequestDetailsPage.tsx

```diff
@@ -168,6 +191,134 @@ function ExceptionRequestDetailsPage() {
                     </Flex>
                 </PageSection>
             </PageSection>
+            <PageSection>
+                <PageSection variant="light">
+                    <Flex direction={{ default: 'column' }}>
+                        <FlexItem>
+                            <Title headingLevel="h2">{cves.length} results found</Title>
+                        </FlexItem>
+                        <FlexItem>
+                            <TableComposable variant="compact">
+                                <Thead noWrap>
+                                    <Tr>
+                                        <Th>{/* Header for expanded column */}</Th>
+                                        <Th>CVE</Th>
+                                        <Th>Images by severity</Th>
+                                        <Th>CVSS</Th>
+                                        <Th>Affected images</Th>
+                                        <Th>First discovered</Th>
+                                    </Tr>
+                                </Thead>
+                                {cves.length === 0 && (
```

Comment:
I don't believe you can have a request without any CVEs, so I might not need to do this 🤔

Reply:
I think you are right, but we should make a note to check for this when we get to the point where we query for the actual data. If somehow an exception had zero cves, when we create the filter and make the request it will retrieve _all_ cves for the table below.